### DISCUSSION TITLE
Task Creation Enabling for Cattr user

### DISCUSSION THREAD

**Discussion by srini4174** [cattr-app/server-application#292] [Q&A]
Hi,

I have successfully installed the Cattr Application on my Ubuntu machine and monitoring the Windows machine of an employee. Now I need help on how to provide permission or enable the employee himself to create a task. Because the task creation is only available for the Administrator in the Cattr dashboard.

Kindly help me resolve the same.

Warm regards,
Srinivasan

**Comment by DanilShapilov** [comment]
@srini4174 Hi. You can make the user a Manager of the specific project.  
But he will also be able to see other people's screenshots inside this specific project.  
![image](https://github.com/user-attachments/assets/e2bc2ebc-b43c-479d-9e79-2f6eae5536c7)

**Reply by srini4174** [reply]
Hi @DanilShapilov, thank you for your suggestion. Is it possible to do changes in the configuration files through cli to make possible the 'user' role to create tasks for himself?

**Reply by DanilShapilov** [reply]
No, it's not possible.

**Reply by srini4174** [reply]
Hi @DanilShapilov, Thank you. I have two more questions. 

1. Is it possible to monitor or view employee attendance details (present and absent status) in addition to the project and time usage reports already available on the dashboard?

2. When I login to the dashboard and when I click on dashboard icon, i am having a pop my in my top right corner of the dashboard as error: The given data failed to pass validation and no reports seen under dashboard. 

Please help on how to resolve these issues.

![image](https://github.com/user-attachments/assets/1624e253-41d4-4695-ad9e-1397b84357e5)

**Reply by DanilShapilov** [reply]
Hi @srini4174  :)
1. In user settings, you can set "Inactivity Time Popup" and if user does not interact with PC, time tracking will be stopped. There was some "app monitoring" which can be enabled in Company settings > Users and works only on windows, but right now I have no idea where it can be viewed :)
2. My guess is that the timezone need to be changed, sorry for this bug, same with Istanbul.
![image](https://github.com/user-attachments/assets/316f6cf6-c10e-4744-b91b-4bb0495b8abb)
